doc/api/fs.markdown

```diff
@@ -116,6 +116,10 @@ Stop watching for changes on the given `fs.FSWatcher`.
 
 Emitted when the ReadStream's file is opened.
 
+### readStream.path
+
+The path to the file the stream will read from.
```

Comment:
Just a bit suggestion: How about changing the `will read from` to `is reading from`, in my opinion, the -ing might be more exact for doing express streams. The same to below `writeStream.path` :-)